### PR TITLE
New Logo and Updated Theme

### PR DESCRIPTION
**Pull request by adambkaplan**
- New Shipwright logo in nav bar
- Added custom navbar to address svg styling
- Added custom CSS styles
- Added custom colors for site color palette, based on logo.

### CODE REVIEW

**Review by qu1queee — APPROVED** [review]
/lgtm

File: assets/icons/logo.svg [hunk 1]
```diff
@@ -1,9 +1,156 @@
...
-  <line style="stroke-linecap: round; fill: rgb(255, 255, 255); stroke: rgb(255, 255, 255); stroke-width: 19px; stroke-dasharray: 30;" x1="137.797" y1="330.007" x2="295.241" y2="330.007"/>
-  <g transform="matrix(6.135525, 0, 0, 6.135525, 175.776047, 233.333344)" style="">
-    <rect fill="none" height="24" width="24"/>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (SaschaSchwarze0) [inline assets/icons/logo.svg:1]: Will this SVG directly go to the webpage? If yes, then we should run it through some minimizer first.

Comment (adambkaplan) [reply assets/icons/logo.svg:1]: It's currently imported via an  tag. I can try in-lining the SVG and use Hugo's built in minifier.

Comment (SaschaSchwarze0) [reply assets/icons/logo.svg]: Just noticed you minified it. Looks better now.

**Review by SaschaSchwarze0 — APPROVED** [review]
/lgtm

File: assets/icons/logo.svg [hunk 2]
```diff
@@ -1,9 +1 @@
...
...n:0;shape-padding:0;text-decoration-color:#000000;text-decoration-line:none;text-decoration-style:solid;text-indent:0;text-orientation:mixed;text-transform:none;white-space:normal"/><path d="m-2.9102 98.074c-1.3225 1.2787-3.7094 1.6179-4.8203 1.5991-1.7723-0.06861-3.5752-0.41414-4.9355-1.5835-1.539 0.88486-3.5261 1.3898-5.0645 1.4181v2c2.3883 0 4.2809-0.58717 4.9668-1.0196 0.68448 0.48247 2.2428 1.0938 4.9688 1.183 2.7734 0.0908 4.3563-0.62773 5.0586-1.1752 0.70092 0.43042 2.6256 1.0118 4.9414 1.0118v-2c-1.7623-0.1341-3.6045-0.47842-5.1152-1.4337z" color="#000000" color-rendering="auto" dominant-baseline="auto" fill="#fff" image-rendering="auto" solid-color="#000000" stop-color="#000000" style="font-feature-settings:normal;font-variant-alternates:normal;font-variant-caps:normal;font-variant-east-asian:normal;font-variant-ligatures:normal;font-variant-numeric:normal;font-variant-position:normal;font-variation-settings:normal;inline-size:0;isolation:auto;mix-blend-mode:normal;shape-margin:0;shape-padding:0;text-decoration-color:#000000;text-decoration-line:none;text-decoration-style:solid;text-indent:0;text-orientation:mixed;text-transform:none;white-space:normal"/></g></g></g></svg>
```
Comment (adambkaplan) [inline assets/icons/logo.svg:1]: @SaschaSchwarze0 I was able to use Inkscape to optimize the svg.

**Pull request merged**